doc/how-to/initialise.rst

```diff
@@ -90,6 +90,39 @@ Complete the following steps to initialise MicroCloud:
 
 See an example of the full initialisation process in the :ref:`Get started with MicroCloud <initialisation-process>` tutorial.
 
+Excluding MicroCeph or MicroOVN from MicroCloud
+-----------------------------------------------
+
+If the MicroOVN or MicroCeph snap is not installed on the system that runs :command:`microcloud init`, you will be prompted with the following question:
+
+  .. terminal::
```

Comment:
```suggestion
If the MicroOVN or MicroCeph snap is not installed on the system that runs :command:`microcloud init`, you will be prompted with the following question::

```

No need for `terminal::` here - we use this when we want to show input and output. For only output, code style is sufficient.